lib/discourse_post_event/rrule_generator.rb

```diff
@@ -3,10 +3,25 @@
 require "rrule"
 
 class RRuleGenerator
-  def self.generate(base_rrule, starts_at, tzid: "UTC")
+  def self.generate(base_rrule, starts_at, tzid: nil)
+    tzid ||= "UTC"
+
     rrule = generate_hash(base_rrule)
+    p rrule
     rrule = set_mandatory_options(rrule, starts_at)
 
+    p rrule
```

Comment:
dis you mean to keep all this debugging?

Reply:
lol oops